### DISCUSSION TITLE
turtle examples on https://schema.org

### DISCUSSION THREAD

**Discussion by justin2004** [schemaorg/schemaorg#3018] [General]
On https://schema.org/Action, for example, there are examples in Microdata, RDFa, JSON-LD but not in Turtle.

Would that be a welcome addition to the website?

**Comment by thadguidry** [comment]
Turtle is just an RDF serialization for easier textual representation for human eyes.  RDFa is also a serialization format for RDF.  You can use a RDFa to Turtle converter.  This also might come in handy, https://github.com/filip26/json-ld-cli
